src/index.html

```diff
@@ -114,11 +114,10 @@
                                     <div class="animated-bullet"></div>
                                 </div>
                                 <div class="bullet-text">
-                                    A programming language optimized for building user
-                                    interfaces with features such as <a href="/null-safety">sound null safety</a>,
-                                    the <a href="/language/collections#spread-operators">spread operator</a> for
-                                    expanding collections, and <a href="/language/collections#control-flow-operators">collection if</a> for
-                                    customizing UI for each platform
+                                    Write safe and concise code using features like <a href="/null-safety">sound null safety</a>,
+                                    <a href="/language/collections#control-flow-operators">collection if</a> for
```

Comment:
I would just drop `for customizing UI for each platform` stuff.

Reply:
Done